webboot/webboot.go

```diff
@@ -84,25 +90,31 @@ var (
 		"arch": &webboot.Distro{
 			"arch/boot/x86_64/vmlinuz",
 			"/arch/boot/x86_64/archiso.img",
-			"memmap=1G!1G earlyprintk=ttyS0,115200 console=ttyS0 console=tty0 root=/dev/pmem0 loglevel=3",
+			*cdl,
```

Comment:
Just change the tinyCoreCmdLine func, for now, and leave this bookmark struct unchanged.